lib/commands/performance.js

```diff
@@ -113,73 +115,81 @@ helpers.getMemoryInfo = async function getMemoryInfo (packageName, dataReadTimeo
     let data = await this.adb.shell(cmd);
     if (!data) throw new Error('No data from dumpsys'); //eslint-disable-line curly
 
-    let totalPrivateDirty, totalPss,
-        nativePrivateDirty, nativePss, nativeHeapSize, nativeHeapAllocatedSize,
-        dalvikPrivateDirty, dalvikPss,
+    let totalPrivateDirty, totalPss, totalRss,
+        nativePrivateDirty, nativePss, nativeHeapSize, nativeHeapAllocatedSize, nativeRss,
+        dalvikPrivateDirty, dalvikPss, dalvikRss,
         eglPrivateDirty, eglPss,
         glPrivateDirty, glPss;
     let apilevel = await this.adb.getApiLevel();
     for (let line of data.split('\n')) {
       let entries = line.trim().split(' ').filter(Boolean);
```

Comment:
does it make sense to rather split by `/\s+/`?

Reply:
also filtering here by `Boolean` might cut zero values, which we probably don't want to happen